static/defaults.config.php

```diff
@@ -562,6 +562,14 @@
 		// Deny undetected languages
 		'relay_deny_undetected_language' => false,
 
+		// relay_language_quality (Integer)
+		// Minimum value for the language detection quality for relay posts. The value must be betweeen 0 and 1.
```

Comment:
I don't think a value that must be "between 0 and 1" should be an integer.
```suggestion
		// relay_language_quality (Float)
		// Minimum value for the language detection quality for relay posts. The value must be between 0 and 1.
```

But maybe it would be simpler to keep the integer constraint and making this setting into a percentage. In this case the setting value must be divided by 100 before being used.
```suggestion
		// relay_language_quality (Integer)
		// Minimum value for the language detection quality for relay posts. The value must be a percentage between 0 and 100.
```

Reply:
I would like to keep the float value, since we use this value in the "language" display:
![grafik](https://github.com/friendica/friendica/assets/844208/68967d2a-6536-4a50-acd6-371cd0787d9d)